python/setup.py

```diff
@@ -310,7 +315,7 @@ def find_version(*filepath):
     "gpustat",
     "grpcio >= 1.28.1",
     "jsonschema",
-    "msgpack >= 0.6.0, < 2.0.0",
+    "msgpack >= 1.0.0, < 2.0.0",
```

Comment:
Can we just replace `install_requires` with a read of the appropriate part of `requirements.txt` instead? so we don't have to keep them in sync.

Reply:
I suppose I could try some minor magic here because the setup.py tends to depend on the whole repo (bazel, et al)

Reply:
Actually, I'll leave the bidirectional sync comments. The separation is much like Robert pointed out (user deps vs dev deps)

Reply:
Is there any disadvantage to reading a requirements file here? I feel like it wouldn't actually cause any issues, and it would prevent the need to keep things in sync, though I might be missing something.

Reply:
Just one more file to install, and the various subpackages